### PR TITLE
Remove Android condition from resetting profile sections on scan

### PR DESCRIPTION
**Pull request by matt-goldman**
Bumped version

### CODE REVIEW

**Comment by matt-goldman** [comment]
DON'T MERGE THIS until this has been merged first: https://github.com/SSWConsulting/SSW.Rewards/pull/386

**Review by bradystroud — APPROVED** [review]

**Pull request merged**
